heudiconv/cli/run.py

```diff
@@ -237,6 +242,9 @@ def process_args(args):
     #
     # Load heuristic -- better do it asap to make sure it loads correctly
     #
+    if not args.heuristic:
+        raise RuntimeError("No heuristic specified - add to arguments and rerun")
+
```

Comment:
Love us great! But if only we could put it in writing (tests) to make sure it stays forever!